src/importexport/midi/internal/midiexport/exportmidi.cpp

```diff
@@ -311,7 +311,7 @@ bool ExportMidi::write(QIODevice* device, bool midiExpandRepeats, bool exportRPN
                                                                                    event.pitch(), 0));
                     }
 
-                    int equivalentStaffIdx = staffIdx;
+                    staff_idx_t equivalentStaffIdx = staffIdx;
                     for (Staff* st : m_score->masterScore()->staves()) {
                         if (staff->id() == st->id()) {
                             equivalentStaffIdx = static_cast <int>(st->idx());
```

Comment:
Oh wait, now we can drop this `static_cast <int>`

Reply:
Done